src/sage/algebras/quantum_clifford.py

```diff
@@ -701,7 +701,7 @@ def _repr_term(self, m):
         """
         p, v = m
 
-        def ppr(i):
+        def ppr(i) -> str:
```

Comment:
/home/runner/work/sage/sage/src/sage/algebras/quantum_clifford.py:704:23 - warning: Function with declared return type "str" must return value on all code paths
  Type "None" cannot be assigned to type "str" (reportGeneralTypeIssues)